examples/csmith/gen.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+
+
+fn_min = 10
+fn_max = 11
+block_min = 4
+block_max = 8 
+seed_min = 0
+num_rand = 50 
+
+# produces programs that we can't handle
+# cmd = ["csmith", "--seed", str(seed), "--max-block-size", str(max_blocksize), "--max-funcs", str(max_funcs), "--output", f"{fname}.c"]
+
+csmith_safe = "--no-arrays --no-bitfields --no-checksum --no-comma-operators --no-longlong --no-int8 --no-uint8 --no-float --no-math64 --no-inline-function --no-safe-math --no-packed-struct --no-pointers --no-structs --no-unions --no-volatile-pointers --no-const-pointers".split(" ")
```

Comment:
I'm curious, what can't we handle about each of these things that have been disabled?

Reply:
It was just an incompleteness in steensgaard wrt MemoryLoads, this list is being overly conservative I'm not sure precisely what its disabling that makes it not generate that case, I'll dig up the example again.

Reply:
Please share whatever case broke the Steensgaard, and maybe file an issue for it.

Reply:
https://github.com/UQ-PAC/BASIL/issues/246